db/migrate/20190925063509_add_detail_to_packages.rb

```diff
@@ -0,0 +1,5 @@
+class AddDetailToPackages < ActiveRecord::Migration
+  def change
+    add_reference :packages, :detail, polymorphic: true, index: true
```

Comment:
often good to add two indexes for composite keys.

1: index on [detail_type, detail_id]
2: index on [detail_id, detail_type]

Reply:
@steveyken , running the migration has already created index.

`add_index "packages", ["detail_type", "detail_id"], name: "index_packages_on_detail_type_and_detail_id", using: :btree`

do you want me to do it seperately ?